### PR TITLE
Add string mulitiplication apoc.text.repeat

### PR DESCRIPTION
**Pull request by tomasonjo**
Fixes https://github.com/neo4j-contrib/neo4j-apoc-procedures/issues/1338

Added string multiplication with `apoc.text.repeat(item,count)`

### CODE REVIEW

**Comment by tomasonjo** [comment]
Have to fix the test error :)

**Comment by tomasonjo** [comment]
IMO is ok now.

File: src/main/java/apoc/text/Strings.java [hunk 1]
```diff
@@ -573,4 +573,10 @@ public String toCypher(@Name("value") Object value, @Name(value = "config",defau
         }
         return null;
     }
+
+    @UserFunction
+    @Description("apoc.text.repeat(item, count) - string multiplication")
+    public String repeat(@Name("item") String item, @Name("count") long count) {
+        return new String(new char[(int) count]).replace("\0", item);
```
Comment (jexp) [inline src/main/java/apoc/text/Strings.java]: better just use a pre-sized StringBuilder and a loop with append.
as the item might be longer than length = 1

**Comment by tomasonjo** [comment]
switched to stringbuilder

**Pull request merged**